client/MainWindow.cpp

```diff
@@ -627,7 +627,8 @@ void MainWindow::enableSign()
 		if( !t.cert().isNull() )
 		{
 			SslCertificate c( t.cert() );
-			signSigner->setText( c.toString( c.showCN() ? "CN (serialNumber)" : "GN SN (serialNumber)" ) );
+			signSigner->setText( signSigner->fontMetrics().elidedText(
+				c.toString( c.showCN() ? "CN (serialNumber)" : "GN SN (serialNumber)" ), Qt::ElideMiddle, 274 ) );
```

Comment:
Please give 274 a name. For example a variable or a define named "maxWidthInPixels" or something similar